pre_commit_hooks/rst_linter.py

```diff
@@ -8,11 +8,12 @@
 def main(argv=None):
     parser = argparse.ArgumentParser()
     parser.add_argument('filenames', nargs='*', help='filenames to check')
+    parser.add_argument('-r', '--raw-enabled', action='store_true', help='Allow the "raw" directive')
```

Comment:
I'm ok this, but `--enable-raw` may be a little bit more _active_ and hence clearer : what do you think?
I'm bikeshedding though :)

Reply:
I actually originally chose `--allow-raw`, but then changed to `--raw-enabled` which is the name of the corresponding setting in `docutils`.

On the other hand, even though `raw_enabled` is the name of the setting internally, we could indeed choose another name for end users.

Do you prefer `--enable-raw` or `--allow-raw`?

Reply:
Given your explanation, even `--raw-enabled` is fine for me :)

`--allow-raw` sounds pretty explicit too though.

Pick your favorite & update the doc accordingly, and I'll merge & release this!

Reply:
Ok, I picked `--allow-raw`. See the latest commit.